src/main/scala/temple/test/internal/ServiceTestUtils.scala

```diff
@@ -151,34 +151,54 @@ object ServiceTestUtils {
       }
       .asJson
 
-  /** Create a new object in a given service, returning the ID field and access token used to make the request */
+  /** Create an object in a given service, returning the unique ID for that object */
+  private def createObject(
+    test: EndpointTest,
+    service: ServiceBlock,
+    serviceName: String,
+    allServices: Map[String, ServiceBlock],
+    baseURL: String,
+    accessToken: String,
+  ): JsonObject = {
+    val requestBody = constructRequestBody(test, service.attributes, allServices, baseURL, accessToken)
+    postRequest(test, s"http://$baseURL/api/${StringUtils.kebabCase(serviceName)}", requestBody, accessToken)
+  }
+
+  /** Create a new object in a given service, returning the ID field */
   def create(
     test: EndpointTest,
     serviceName: String,
     allServices: Map[String, ServiceBlock],
     baseURL: String,
     accessToken: String,
-  ): CreateResponse = {
+  ): String = {
     val service = allServices.getOrElse(serviceName, test.fail(s"service $serviceName does not exist"))
-    // If this service is an auth service, the same access token cannot be used twice, so make a new one to be safe...
-    // This is so that services that reference 2+ auth'd services can be successfully tested
-    // TODO: refactor for central auth method
-    val newAccessToken = service.lookupLocalMetadata[Metadata.ServiceAuth].fold(accessToken) { _ =>
-      ServiceTestUtils.getAuthTokenWithEmail(serviceName, baseURL)
+
+    // If this service is an auth service, an access token can only create one entity
+    val createJSON = service.lookupLocalMetadata[Metadata.ServiceAuth] match {
+      case Some(_) =>
+        // First check if that single entity already exists
+        val response = Http(s"http://$baseURL/api/${StringUtils.kebabCase(serviceName)}")
+          .method("GET")
+          .header("Authorization", s"Bearer $accessToken")
+          .asString
+        response.code match {
+          case HttpURLConnection.HTTP_MOVED_TEMP =>
+            // Already exists, so perform a GET on it now the ID is known
+            val url = response.header("Location").getOrElse(test.fail("302 response without Location header"))
+            getRequest(test, s"http://$url", accessToken)
```

Comment:
This feels like a broken url format if you have to stick `http://` onto it

Reply:
Will update the location header in a separate PR.